edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.Test/DockerConfigTest.cs

```diff
@@ -25,6 +28,10 @@ public class DockerConfigTest
 
         static readonly DockerConfig Config10 = new DockerConfig("image1:42", @"{""Env"": [""k11=v11"", ""k22=v22""], ""HostConfig"": {""PortBindings"": {""43/udp"": [{""HostPort"": ""43""}]}}}");
         static readonly DockerConfig Config11 = new DockerConfig("image1:42", @"{""Env"": [""k33=v33"", ""k44=v44""], ""HostConfig"": {""PortBindings"": {""43/udp"": [{""HostPort"": ""43""}]}}}");
+        static readonly DockerConfig Config12 = new DockerConfig("image1:42", string.Empty);
```

Comment:
Also suggest adding a test for unknown image for completeness.

Reply:
Sure..